benchmarks/libxml2-v2.9.2/build.sh

```diff
@@ -26,6 +26,8 @@ build_lib() {
 }
 
 get_git_tag https://gitlab.gnome.org/GNOME/libxml2.git v2.9.2 SRC
+get_git_revision https://github.com/google/afl f10d601b3f3026461c669251696e6f1328ce6c00 afl
```

Comment:
nit: can we just curl/wget https://github.com/google/AFL/blob/debe27037b9444bbf090a0ffbd5d24889bb887ae/dictionaries/xml.dict
Probably slightly faster than cloning whole repo.

Reply:
Done!